### PR TITLE
Add libconda

### PR DESCRIPTION
**Pull request by jakirkham**
Adds a recipe for `libconda` written from scratch. Feedback and maintainers welcome.

cc @conda-forge/core @ilanschnell

### CODE REVIEW

File: recipes/libconda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set version = "4.0.0" %}
+{%set sha256 = "8cc07dc407527fdb4d028c8caf67757f6df08d045aa03e1ecd9796aa0489a3a2" %}
+
+package:
+  name: libconda
+  version: {{ version }}
+
+source:
+  fn: libconda-{{ version }}.tar.gz
+  url: https://github.com/ilanschnell/libconda/archive/{{ version }}.tar.gz
```
Comment (pelson) [inline recipes/libconda/meta.yaml]: Can be updated to the conda organisation at this point.

**Comment by pelson** [comment]
No complaints from me, though it would be healthy if @ilanschnell were able to help maintain.

File: recipes/libconda/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/libconda/meta.yaml]: Thanks for the heads-up. Have updated.

**Comment by ilanschnell** [comment]
libconda will not require a lot of maintaining, because the point of the library is stability, and as few releases as possible.  In fact, I'm not planning any new releases at all at this point :-)

File: recipes/libconda/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/libconda/meta.yaml]: Now I remember why I did this. There is no tag on the org repo. I have added an issue ( https://github.com/conda/libconda/issues/1 ) upstream for this change.

Comment (jakirkham) [inline recipes/libconda/meta.yaml]: This has now been addressed. Will restart the CIs.

**Comment by jakirkham** [comment]
Then it should be fine to a maintainer, right? 😄 (sorry twisting your arm just a little)

**Comment by ilanschnell** [comment]
I'm already maintaining the recipe here:
https://github.com/ContinuumIO/anaconda-recipes/tree/master/libconda

**Pull request merged**